bob/src/cluster/quorum.rs

```diff
@@ -272,21 +272,130 @@ impl Cluster for Quorum {
     }
 
     async fn exist(&self, keys: &[BobKey]) -> Result<Vec<bool>, Error> {
```

Comment:
Function is too large and should be decomposed

Reply:
Done: [2bd9373](https://github.com/qoollo/bob/pull/634/commits/2bd9373699069170d8982387dc7806df38e3c957)